app/helpers/patient_query_helper.rb

```diff
@@ -180,43 +180,44 @@ def sort(patients, order, direction)
 
     case order
     when 'name'
-      patients = patients.order(last_name: dir).order(first_name: dir)
+      patients = patients.order(last_name: dir).order(first_name: dir).order(id: dir)
```

Comment:
Ya, I was wavering on whether or not to add this one too, so that sounds reasonable to me. I will remove the secondary ordering for this line that sorts based on the monitoree's name in my next commit.

Reply:
Oh sorry I mean all of the order changes in this file I don't think are necessary, not just this one.

Reply:
The main reason I thought including this precise ordering was important was because we sometimes keep the page number sticky, and that would only make sense if the user sees the same monitorees on that sticky page number that they were viewing before.

To help clarify, here is the use case I am envisioning: A users is on page 3 of the Symptompatic line list on the Exposure workflow and they have sorted the monitorees based on Jurisdiction. Let's say they are currently in the middle of checking through all monitorees in USA, State 1 and there are a few more pages of monitorees in that jurisdiction. The user selects monitoree A to look at their profile, and when done, goes back to the Symptomatic line list on the Exposure dashboard. According to how the page sticky setting works, the user will be brought back to page 3 as expected. BUT since the sorted ordering won't be precisely the same as the previous ordering, the user could see a completely different set of monitorees and loose track of the monitorees they have versus haven't followed up with yet if they were simply running down the list.

So I think having this precise ordering is important if the page number is ever sticky. If the page number isn't sticky, I would agree that the precise orderings are not important. 

After considering this use case, feel free to let me know your thoughts on how I should proceed. I'd be happy to explain further if need be.

Reply:
That use case makes sense, especially because when sorting by things like jurisdiction it's expected there will be a lot of duplicate values. Given this need I think we should do two things then:

1. Do performance regression testing on these queries. How much slower are these sorts on hundreds of thousands of monitorees?
2. Clean this up so the `id` is a second argument in `order` rather than in an additional `order` call (ex: `patients.order(last_name: dir, id: dir)` rather than `patients.order(last_name: dir).order(id: dir)`. Small thing, but helps reduce code.

Reply:
Results from some performance regression tests for those sort queries. Looks like the impact is negligible so unless there are any other concerns, I'll move forward to keep the secondary ID sorts in the PR.

![image](https://user-images.githubusercontent.com/42656458/107057360-5f3a8b80-67a1-11eb-85b8-7f7ab5f7d304.png)

Reply:
Thanks - it's looking like it'll be fine!